libfreerdp/primitives/test/TestPrimitivesAlphaComp.c

```diff
@@ -179,8 +178,6 @@ static int test_alphaComp_speed(void)
 	for (i = 0; i < sizeof(src2) / 4; ++i)
 		*ptr++ |= 0xFF000000U;
 
-	memset(dst1, 0, sizeof(dst1));
```

Comment:
The first `memset` removal was fine. But does this apply to this one too? I haven't specifically looked into the function calls above, whether they change `dst1`, but if they do, the behaviour of this test changes here.

Reply:
yes, the loop uses `ptr = (UINT32*)src2;` so `dst1` is never touched before